extension/src/test/suite/experiments/index.test.ts

```diff
@@ -1464,59 +1465,24 @@ suite('Experiments Test Suite', () => {
       await experiments.isReady()
       await experiments.showWebview()
 
-      expect(messageSpy).to.be.calledWithMatch({
-        rows: [
-          {
-            branch: 'main',
-            displayColor: undefined,
-            id: EXPERIMENT_WORKSPACE_ID,
-            label: EXPERIMENT_WORKSPACE_ID,
-            selected: false,
-            starred: false
-          },
-          {
-            branch: 'main',
-            displayColor: undefined,
-            id: 'testBranch',
-            label: 'testBranch',
-            selected: false,
-            starred: false,
-            subRows: [
-              {
-                branch: 'main',
-                displayColor: undefined,
-                description: '[exp-1]',
-                id: 'exp-1',
-                label: '111111',
-                params: { 'params.yaml': { test: 2 } },
-                selected: false,
-                starred: false
-              },
-              {
-                branch: 'main',
-                displayColor: undefined,
-                description: '[exp-2]',
-                id: 'exp-2',
-                label: '222222',
-                params: { 'params.yaml': { test: 1 } },
-                selected: false,
-                starred: false
-              },
-              {
-                branch: 'main',
-                displayColor: undefined,
-                description: '[exp-3]',
-                id: 'exp-3',
-                label: '333333',
-                params: { 'params.yaml': { test: 3 } },
-                selected: false,
-                starred: false
-              }
-            ]
+      const getIds = (rows: Commit[]) =>
```

Comment:
[F] I always run into/get annoyed by this test. It was previously testing more than it needed to. It is not testing only the functionality that it needs to.